modal.js

```diff
@@ -0,0 +1,125 @@
+const recentWorkContainer = document.getElementById('multi-post-grid');
+const recentWork = [
+  {
+    picFile: 'work1.png',
+    project_title: 'Multi-Post Stories',
+    description: 'A daily section of privately personalized reads; no accounts or sign-ups required. has been the industry&#x27;s standards dummy text ever since the 1500s, when an unknown printer took a standard dummy text',
+    skill: ['css', 'html', 'bootstrap', 'Ruby'],
+  },
+  {
+    picFile: 'work2.png',
+    project_title: 'Multi-Post Stories',
+    description: 'A daily section of privately personalized reads; no accounts or sign-ups required. has been the industry&#x27;s standards dummy text ever since the 1500s, when an unknown printer took a standard dummy text',
+    skill: ['css', 'html', 'bootstrap', 'Ruby'],
+  },
+  {
+    picFile: 'work3.png',
+    project_title: 'Multi-Post Stories',
+    description: 'A daily section of privately personalized reads; no accounts or sign-ups required. has been the industry&#x27;s standards dummy text ever since the 1500s, when an unknown printer took a standard dummy text',
+    skill: ['css', 'html', 'bootstrap', 'Ruby'],
+  },
+  {
+    picFile: 'work4.png',
+    project_title: 'Multi-Post Stories',
+    description: 'A daily section of privately personalized reads; no accounts or sign-ups required. has been the industry&#x27;s standards dummy text ever since the 1500s, when an unknown printer took a standard dummy text',
+    skill: ['css', 'html', 'bootstrap', 'Ruby'],
+  },
+];
+const loadRecentWork = () => {
+  let cardRecentWork = '';
+  for (let recentWorkIndex = 0; recentWorkIndex < recentWork.length; recentWorkIndex += 1) {
+    const card = `<div class="multi-post multi-post-${recentWorkIndex + 1}">
+        <img class="image-placeholder image-${recentWorkIndex + 1}" src="./image/Recent-work/${recentWork[recentWorkIndex].picFile}" alt="work-image"></img>
+        <div class="flex-container flex-container-${recentWorkIndex + 1}">
+            <h2 class="multi-post-header">${recentWork[recentWorkIndex].project_title}</h2>
+            <p class="multi-post-paragraph">${recentWork[recentWorkIndex].description}</p>
+            <ul class="multi-post-skill">
+                <li class="post-skill">
+                    <p class="skill-lang">${recentWork[recentWorkIndex].skill[0]}</p>
+                </li>
+                <li class="post-skill">
+                    <p class="skill-lang">${recentWork[recentWorkIndex].skill[1]}</p>
+                </li>
+                <li class="post-skill">
+                    <p class="skill-lang">${recentWork[recentWorkIndex].skill[2]}</p>
+                </li>
+                <li class="post-skill">
+                    <p class="skill-lang">${recentWork[recentWorkIndex].skill[3]}</p>
+                </li>
+            </ul>
+            <button type="button" class="see-project">See Project</button>
+        </div>
+        </div>`;
```

Comment:
- [ ] Kindly make sure all your cards are showing the popup as the 4th one in your portfolio can not show the popup when the user clicks the **See Project** button.